<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LTB0216523-MAC-2.local" failures="0" tests="15" name="com.airtel.RestAssured.Adtech.Tests.billing.ReceiptTest" time="4.421" errors="1" timestamp="18 May 2022 07:45:42 GMT" skipped="14">
  <testcase classname="com.airtel.RestAssured.Adtech.Tests.billing.ReceiptTest" name="setEnvironment" time="4.419">
    <error message="Cannot invoke &quot;String.toCharArray()&quot; because &quot;val&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "String.toCharArray()" because "val" is null
	at java.base/java.math.BigDecimal.<init>(BigDecimal.java:896)
	at com.airtel.adtech.manager.BillingAdtechApiManager.<init>(BillingAdtechApiManager.java:72)
	at com.airtel.RestAssured.Adtech.Tests.billing.ReceiptTest.setEnvironment(ReceiptTest.java:99)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:166)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)
]]>
    </error>
  </testcase> <!-- setEnvironment -->
  <testcase classname="com.airtel.RestAssured.Adtech.Tests.billing.ReceiptTest" name="validateInitiatePaymentSuccess" time="0.000">
    <skipped/>
  </testcase> <!-- validateInitiatePaymentSuccess -->
  <testcase classname="com.airtel.RestAssured.Adtech.Tests.billing.ReceiptTest" name="validateInitiatePaymentFailure" time="0.000">
    <skipped/>
  </testcase> <!-- validateInitiatePaymentFailure -->
  <testcase classname="com.airtel.RestAssured.Adtech.Tests.billing.ReceiptTest" name="initiatePaymentwithInvalidAmount" time="0.000">
    <skipped/>
  </testcase> <!-- initiatePaymentwithInvalidAmount -->
  <testcase classname="com.airtel.RestAssured.Adtech.Tests.billing.ReceiptTest" name="initiatePaymentOnPartialPaidInvoice" time="0.000">
    <skipped/>
  </testcase> <!-- initiatePaymentOnPartialPaidInvoice -->
  <testcase classname="com.airtel.RestAssured.Adtech.Tests.billing.ReceiptTest" name="initiatePaymentWithInvalidEvent" time="0.000">
    <skipped/>
  </testcase> <!-- initiatePaymentWithInvalidEvent -->
  <testcase classname="com.airtel.RestAssured.Adtech.Tests.billing.ReceiptTest" name="initiatePaymentWithMultipleInvoices" time="0.000">
    <skipped/>
  </testcase> <!-- initiatePaymentWithMultipleInvoices -->
  <testcase classname="com.airtel.RestAssured.Adtech.Tests.billing.ReceiptTest" name="verifyValidateAPITwice" time="0.001">
    <skipped/>
  </testcase> <!-- verifyValidateAPITwice -->
  <testcase classname="com.airtel.RestAssured.Adtech.Tests.billing.ReceiptTest" name="initiatePaymentOnProceesedStatus" time="0.000">
    <skipped/>
  </testcase> <!-- initiatePaymentOnProceesedStatus -->
  <testcase classname="com.airtel.RestAssured.Adtech.Tests.billing.ReceiptTest" name="validateInvoiceAndReceipt" time="0.001">
    <skipped/>
  </testcase> <!-- validateInvoiceAndReceipt -->
  <testcase classname="com.airtel.RestAssured.Adtech.Tests.billing.ReceiptTest" name="validateInvoiceReceiptWithLessInvoiceAmount" time="0.000">
    <skipped/>
  </testcase> <!-- validateInvoiceReceiptWithLessInvoiceAmount -->
  <testcase classname="com.airtel.RestAssured.Adtech.Tests.billing.ReceiptTest" name="verifyReceiptsforMultipleInvoices" time="0.000">
    <skipped/>
  </testcase> <!-- verifyReceiptsforMultipleInvoices -->
  <testcase classname="com.airtel.RestAssured.Adtech.Tests.billing.ReceiptTest" name="verifyReceiptsforMultipleInvoiceswithPendingReceipts" time="0.000">
    <skipped/>
  </testcase> <!-- verifyReceiptsforMultipleInvoiceswithPendingReceipts -->
  <testcase classname="com.airtel.RestAssured.Adtech.Tests.billing.ReceiptTest" name="verifyReceiptswithGreaterAmountThanInvoice" time="0.000">
    <skipped/>
  </testcase> <!-- verifyReceiptswithGreaterAmountThanInvoice -->
  <testcase classname="com.airtel.RestAssured.Adtech.Tests.billing.ReceiptTest" name="duplicateEvent" time="0.000">
    <skipped/>
  </testcase> <!-- duplicateEvent -->
</testsuite> <!-- com.airtel.RestAssured.Adtech.Tests.billing.ReceiptTest -->
